runtime/doc/options.txt

```diff
@@ -6926,4 +6926,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	When negative, all redrawn characters cause a delay, even if the
 	character already was displayed by the UI.  For debugging purposes.
 
+	Once writedelay is set to some postitive or negative integer, it
+	can be reset to 0 by triggering two interrupts (Hitting <C-C> two
+	times)
```

Comment:
Too verbose. Instead:

    <C-C><C-C> resets writedelay to 0.

Reply:
Or `Using CTRL-C twice (during one main loop iteration) will reset it.`.